unit_tests/common/test_cuda_is_ptr.cu

```diff
@@ -31,18 +31,6 @@ TEST(test_is_gpu_ptr, cuda_ptr_is_cuda_ptr) {
     EXPECT_EQ(cudaSuccess, cudaGetLastError());
 }
 
-TEST(test_is_gpu_ptr, cudaMallocHost_ptr_is_not_cuda_ptr) {
-    auto testee = std::shared_ptr<double>(
-        []() {
-            double *res;
-            GT_CUDA_CHECK(cudaMallocHost(&res, sizeof(double)));
```

Comment:
cuda/hipMallocHost is not supported by HIP, but test seems irrelevant for us anyway, so I removed it.